Flowpack/media-ui
# FEATURE: add and delete AssetCollections and Tags

Resolves: https://github.com/Flowpack/media-ui/issues/24

TODO:
- [x] get local cache to update
- [x] optimistic update?
- [x] create collection modal
- [x] create tag modal
Maybe later:
- [ ] expand collection when creating a tag
- [ ] setFocus doesn't work in modals for some reason

**What I did**

- Identify AssetCollections by id instead of title.
- Add and delete operations for AssetCollections
- Add and delete tags

**How to verify it**

Try to create and remove asset collections and tags. Make sure the state updates correctly, including optimistic update.
